dataset_schema.json

```diff
@@ -204,5 +204,5 @@
         }
     },
     "additionalProperties": false,
-    "required" : [ "title", "types", "creators", "licenses", "description", "keywords", "version", "distributions" ]
+    "required" : [ "title", "types", "creators", "licenses", "description", "keywords", "version", "distributions", "privacy" ]
```

Comment:
privacy is not a required field I think. @emmetaobrien

Reply:
It is not currently, and we can't enforce it globally because of not being able to modify crawled data.  The restrictions here should be on the content of the privacy field **IF** it exists; I think the appropriate counterpart to that is to make it a required field in the GUI so that people cannot generate `DATS.json` files through the interface without having a valid `privacy` value, rather than trying to control it in the schema.

Reply:
@emmetaobrien Indeed, in the GUI code, this has been modified and merged last week. All that needs to be done is pull that change to the conp-portal repo which I will try to figure out this week.

Reply:
Excellent, thank you.